docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -18097,6 +18097,284 @@
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-HdInsight-MinimalTlsVersion"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-HdInsight-NetworkProperties",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Enforces private link enablement for hd insight clusters.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "HDInsight"
+            },
+            "parameters": {
+              "effect": {
+                "type": "String",
+                "metadata": {
+                  "displayName": "Effect",
+                  "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                  "Audit",
+                  "Disabled",
+                  "Deny"
+                ],
+                "defaultValue": "Deny"
+              }
+            },
+            "policyRule": {
+              "if": {
+                "allOf": [
+                  {
+                    "field": "type",
+                    "equals": "Microsoft.HDInsight/clusters"
+                  },
+                  {
+                    "anyOf": [
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.resourceProviderConnection",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.resourceProviderConnection",
+                        "notEquals": "Outbound"
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.privateLink",
+                        "exists": false
+                      },
+                      {
+                        "field": "Microsoft.HDInsight/clusters/networkProperties.privateLink",
+                        "notEquals": "Enabled"
+                      }
+                    ]
+                  }
+                ]
+              },
+              "then": {
+                "effect": "[[parameters('effect')]"
+              }
+            }
+          },
+          "name": "Deny-HdInsight-NetworkProperties"
+        },
+        {
+          "properties": {
+            "displayName": "Deny-HdInsight-Sku",
+            "policyType": "Custom",
+            "mode": "Indexed",
+            "description": "Enforces certain sku's for hd insight clusters.",
+            "metadata": {
+              "version": "1.0.0",
+              "category": "Budget"
```

Comment:
Should this be `HDInsight` as per the others to keep them all grouped together nicely?

Reply:
But this is mostly for cost management. Feel free to change it.

Reply:
Is there a technical reason (security related etc.) why one would dictate sku's for HDInsights clusters from a platform perspective?